bin/util/transifex.js

```diff
@@ -72,15 +72,22 @@ const parseVars = (pluralForm) => {
 class PluralForms {
   static empty(length) { return new PluralForms(new Array(length).fill('')); }
 
-  static fromVueI18n(message) {
+  static fromVueI18n(path, message) {
     const forms = message.split(' | ');
     if (forms.length > 2)
       logThenThrow(message, 'a pluralized message must have exactly two forms');
 
     for (const form of forms) {
       if (form.includes('|')) logThenThrow(message, 'unexpected |');
-      if (/(^\s|\s$|\s\s)/.test(form))
-        logThenThrow(message, 'unexpected white space');
+
+      const badWhitespace = form.match(/^\s+|\s+$|\s\s+/);
+      if (badWhitespace) {
+        const badLength = badWhitespace[0].length;
+        console.error(`unexpected white space in translation string '${path}':`); // eslint-disable-line no-console
+        console.error(`  [${message}]`); // eslint-disable-line no-console
+        console.error(`  [${''.padStart(badLength, '^').padStart(badWhitespace.index + badLength, ' ').padEnd(message.length, ' ')}]`); // eslint-disable-line no-console
+        throw new Error(`unexpected whitespace in message '${path}' at index ${badWhitespace.index} ("${message}")`);
```

Comment:
I feel like this doesn't fully account for the difference between `message` and `form`. As an example, for the pluralized message `App User | App  Users`, the second plural form is the one with multiple spaces. Here, the entire message would be shown (both plural forms), but then the index from the match on only the second form would be used to position the ^^^ and describe the index of the unexpected whitespace.

How about something like:

```js
const error = forms.length === 1
  ? `unexpected white space in message '${path}'`
  : `unexpected white space in plural form of message '${path}'`;
console.error(error); // eslint-disable-line no-console
const badLength = badWhitespace[0].length;
console.error(`  [${form}]`); // eslint-disable-line no-console
console.error(`  [${''.padStart(badLength, '^').padStart(badWhitespace.index + badLength, ' ').padEnd(message.length, ' ')}]`); // eslint-disable-line no-console
if (forms.length !== 1)
  console.error(`The entire message is: [${message}]`); // eslint-disable-line no-console
throw new Error(`${error} at index ${badWhitespace.index} ("${form}")`);
```

Reply:
Good catch.

I've updated to consider the index of the form.  Some example errors:

#### single-value
```
unexpected white space in translation string 'datasets':
  [Data  sets]
  [    ^^    ]
...
Error: unexpected whitespace in message 'datasets' at index 4 ("Data  sets")
```

#### singular of multi-value
```
unexpected white space in translation string 'appUser[0]':
  [App  User]
  [   ^^    ]
...
Error: unexpected whitespace in message 'appUser[0]' at index 3 ("App  User | App  Users")
```

#### plural of multi-value
```
unexpected white space in translation string 'appUser[1]':
  [App  Users]
  [   ^^     ]
...
Error: unexpected whitespace in message 'appUser[1]' at index 3 ("App User | App  Users")
```